galaxykubeman/values.yaml

```diff
@@ -7,6 +7,18 @@ configs: {}
 secrets: {}
   # db-password: changeme
 
+persistence: {}
```

Comment:
as we spoke about offline, leo will be creating two disks in order to avoid the performance issues mentioned during the leo/galaxy sync up meeting last week. one disk for nfs and the other for postgres :)

Reply:
The commented example below is how we'd set the values for any number of disks, so for two disks it's just copy-pasting the bit and changing the top key. The templating will loop through all keys and create a PV/PVC pair for each.